sys-libs/mesa/patches/mesa-18.2.0rc2.patchset

```diff
@@ -0,0 +1,509 @@
+diff --git a/src/Makefile.am b/src/Makefile.am
```

Comment:
It is still not exported from haikuporter, it should get a git-formal-patch header.

Reply:
Please, do not take-over patches, you have to keep the original headers (author, commit message, etc).
Do not merge patches into one big patch.